### DISCUSSION TITLE
Enable included datapacks

### DISCUSSION THREAD

**Discussion by Elmardus** [itzg/docker-minecraft-server#1796] [Q&A]
Snapshot 22W42A has 'built-in' datapacks which include experimental features. How can these datapacks be enabled?
https://www.minecraft.net/en-us/article/minecraft-snapshot-22w42a

**Comment by itzg** [comment]
I'll get the new properties settable via env vars:

https://www.minecraft.net/en-us/article/minecraft-snapshot-22w42a#:~:text=Added%20new%20fields,not%20auto%2Denable

**Comment by itzg** [comment]
Tracking in https://github.com/itzg/docker-minecraft-server/issues/1797
